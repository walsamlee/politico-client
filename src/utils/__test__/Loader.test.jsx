import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../Loader';

describe('<spinLoader />', () => {
  it('should render spinLoader component', () => {
    const wrapper = shallow(
      <Loader />,
    );
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('div'));
  });
});
