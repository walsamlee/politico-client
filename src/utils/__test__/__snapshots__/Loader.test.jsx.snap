// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<spinLoader /> should render spinLoader component 1`] = `ShallowWrapper {}`;
